docs/admin/install.rst

```diff
@@ -0,0 +1,243 @@
+- A working computer (Linux is recommended and assumed in this guide) to use for verification and creation of the Qubes installation medium.
+- A password manager or other system to generate and store strong passphrases for Qubes FDE and user accounts.
+
+A basic knowledge of the Qubes OS is helpful. 
+
+
+Apply BIOS updates and check settings
+-------------------------------------
+Before beginning the Qubes installation, make sure that your Qubes-compatible computer's BIOS is updated to the latest available version. Instructions on how to do so will vary by manufacturer or model and can usually be found on their respective support sites, for example `https://pcsupport.lenovo.com/ <https://pcsupport.lenovo.com/>`_ in the case of Lenovo.
+
+Once the BIOS is up-to-date, boot into the BIOS setup utility and verify that:
+
+- clock settings are correct
+- virtualization support is enabled
+  - for Intel-based devices, **Intel VT-d** and **Intel VT-x** should be enabled
+  - for AMD-based devices, **AMD-VI** and **AMD-V** should be enabled
+- SecureBoot is disabled.
+
+If the Qubes hardware compatibility list entry for your computer recommends the use of Legacy Mode for boot, change that setting in the BIOS as well, otherwise choose UEFI or leave it on the default.
+
+Download and verify QubesOS
+---------------------------
+On the working computer, download the latest stable Qubes OS ISO (``4.0.3`` at time of writing) from `https://www.qubes-os.org/downloads/ <https://www.qubes-os.org/downloads/>`_.
+
+Follow the linked instructions to `verify the ISO <https://www.qubes-os.org/security/verifying-signatures/#how-to-verify-qubes-iso-signatures>`_.
+
+Once you've verified the ISO, copy it to your installation medium - for example, if using Linux and a USB stick, using the command:
+
+.. code-block:: sh
+ 
+  sudo dd if=Qubes-R4.0.3-x86_64.iso of=/dev/sdX bs=1048576 && sync
+
+where ``if`` is set to the path to your downloaded ISO file and ``of`` is set to
+the block device corresponding to your USB stick. Make sure to verify that you have the correct device name using, for example, the ``lsblk`` command.
+
+
+Install QubesOS
+---------------
+To begin the Qubes installation, connect the Qubes install USB to your target computer and boot from it. You may need to bring up a a boot menu at startup to do so - on Lenovo laptops, for example, you can do so by pressing **F12** on boot.
+
+Follow the `installation documentation <https://www.qubes-os.org/doc/installation-guide/>`_ to install Qubes on your computer, ensuring that you:
+
+- Use all available storage space for the installation (as the computer should be dedicated to SecureDrop Workstation).
+- Set a strong FDE passphrase - a 6-word Diceware passphrase is recommended.
+- Create an administrative account with a strong password. This documentation assumes the username is ``user``.
```

Comment:
I think we should discuss the value of a "strong password" for the administrative password in the context of our threat model. Once you have the FDE passphrase, you can boot into single user mode and exfiltrate secrets. A complex password for the main Qubes account adds significant burden to the admin user who will have to type this password in e.g. from a phone, or memorize it. Does it add commensurate value?

Reply:
:heavy_check_mark:

Reply:
TBH I think it adds more opportunity for error to say to users "this needs a password that's strong, this not so much", as they now have to decide what that means. Given password managers exist and we encourage their use, I would suggest that we keep things binary; either we recommend a strong passphrase, or in situations where, for example, stuff is behind an FDE passphrase, we recommend no passphrase.

Reply:
In this case, as we don't know if folks will treat the workstation like the SVS physical-security-wise, IMO it's probably worth setting a passphrase to guard against shoulder surfers or unattended machines. (This does not guard against the workstation storage being read if left unlocked.)

Reply:
Yeah a passphrase that's easy to type would work, if there's no character limit. Whatever language we use should be explicit enough to avoid them picking a password that's impossible or overly laborious to type since they'll have to unlock screensaver with it repeatedly.

Reply:
Diceware passwords are supposed to be less hassle to type.

Reply:
OK updated note accordingly